docs/testing.rst

```diff
@@ -381,6 +381,67 @@ functionality, the AioHTTPTestCase is provided::
    Handles executing an asynchronous function, using
    the :attr:`AioHTTPTestCase.loop` of the :class:`AioHTTPTestCase`.
 
+Patching unittest test cases
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Patching test cases is tricky, when using python older than 3.8 ``'patch'`` does not behave as it has to.
+We recommend using asynctest_ that provides a patch method that is capable of creating
+a magic mock that supports async. It can be used with decorator as well as with context manager::
+
```

Comment:
Let's highlight the important lines:
```suggestion
a magic mock that supports async. It can be used with a decorator as well as with a context manager:

.. code-block:: python
   :emphasize-lines: 1,37,46
```
(https://www.sphinx-doc.org/en/1.8/usage/restructuredtext/directives.html#directive-code-block)

Reply:
this highlight does not look good, it seems to me that it only highlights spaces

Reply:
<img width="662" alt="Screen Shot 2020-04-20 at 8 41 14 PM" src="https://user-images.githubusercontent.com/5825811/79782158-4f9fa280-8347-11ea-8086-39301eeee734.png">

Reply:
Could you plz post a screenshot?

Reply:
Alright, it looks like the Sphinx theme doesn't account for highlighting. So the theme itself should be fixed but this PR is correct, let's keep it.